addon/components/rdf-input-fields/concept-scheme-multi-select-checkboxes.js

```diff
@@ -50,6 +51,11 @@ export default class RDFInputFieldsConceptSchemeMultiSelectCheckboxesComponent e
     const { sourceGraph, metaGraph } = this.args.graphs;
     const path = this.args.field.rdflibPath;
     const options = this.args.field.options;
+
+    if (!hasValidFieldOptions(this.args.field, ['conceptScheme', 'orderBy'])) {
```

Comment:
I saw on two places that the `orderBy` is also present in the ttl files, ref: [subsidie-beheer](https://github.com/lblod/app-digitaal-loket/blob/fab8316a51e73e16f3af4a9218270dc3b63b74a6/config/subsidy-application-management/forms/e-inclusion/proposal/versions/20220804090000/form.ttl#L247) but this is not required for showing the component correctly.

Should we add it here as a required property?

Reply:
I wouldn't add it, no. The component seems to consider it optional.